src/validation-helpers.js

```diff
@@ -61,3 +61,20 @@ export function validateArray(propertyName, arr, cbk) {
     cbk(val, `${propertyName}[${index}]`);
   });
 }
+
+export function validateContainerEl(propertyName, containerEl) {
```

Comment:
I may have missed it but I don't see a test for this new validation method

Reply:
Good catch - udpated